src/gt4py/next/program_processors/runners/gtfn.py

```diff
@@ -165,13 +167,18 @@ class Params:
             ),
             name_cached="_cached",
         )
+        force_temporaries = factory.Trait(
```

Comment:
```suggestion
        use_temporaries = factory.Trait(
```
This enables them altogether so before we introduce a wrong name that is user facing it would be great if you could also rename `LiftMode.FORCE_TEMPORARIES` to `USE_TEMPORARIES`. I should have renamed this when I introduced the extraction heuristics.

Reply:
👍